Source/UI/NewUI/CommonUI/View/TextFieldView.swift

```diff
@@ -137,7 +154,17 @@ extension TextFieldView {
         NSLayoutConstraint.activate([
             headerLabel.topAnchor.constraint(equalTo: topAnchor),
             headerLabel.leadingAnchor.constraint(equalTo: leadingAnchor),
-            headerLabel.trailingAnchor.constraint(equalTo: trailingAnchor)
+        ])
+    }
+
+    private func setupMandatoryLabel() {
+        guard let mandatoryLabel = mandatoryLabel else { return }
+        guard let headerLabel = headerLabel else { return }
+        addSubview(mandatoryLabel)
+        NSLayoutConstraint.activate([
+            mandatoryLabel.topAnchor.constraint(equalTo: topAnchor),
```

Comment:
I am assuming leadingAnchor is not required here as the super view does that ?. I am concerned if this mandatoryLabel will support RTL.

Reply:
Tested and working fine
<img width="362" alt="Screenshot 2022-06-09 at 2 31 33 pm" src="https://user-images.githubusercontent.com/102961713/172859355-fea04f55-2e19-4b20-b7c3-e1be7981a04b.png">